lib/service/services_cli.rb

```diff
@@ -142,14 +142,23 @@ def stop(targets, verbose: false)
           next
         end
 
-        puts "Stopping `#{service.name}`... (might take a while)"
+        systemctl_args = System.systemctl_args
+        if no_wait
+          systemctl_args << "--no-block"
```

Comment:
It doesn't look like the `systemctl_args` variable is actually used anywhere in this method outside of lines 145 and 147. I assume it's supposed to be used on line 154 as well.

Reply:
@apainintheneck Whoops, opened PR in https://github.com/Homebrew/homebrew-services/pull/596